lib/click.ts

```diff
@@ -1,6 +1,7 @@
 import { InsightsEvent } from "./_sendEvent";
 
 export interface InsightsSearchClickEvent {
+  eventName: string;
```

Comment:
This was missing. It could be a separate PR on its own, but I just added it here.

Reply:
I think it's fine to have it here since it's still related to typescript